src/ring_theory/jacobson_ideal.lean

```diff
@@ -86,6 +97,110 @@ theorem mem_jacobson_iff {I : ideal R} {x : R} :
         neg_mul_eq_neg_mul_symm, neg_mul_eq_mul_neg, mul_comm x y]; exact M.mul_mem_right hy)
     (him hz)⟩
 
+/-- An ideal equals its jacobson radical iff it is the intersection of a set of maximal ideal.
+Allowing the set to include ⊤ is equivalent, and is included only to simplify some proofs. -/
+theorem eq_jacobson_iff_Inf_maximal {I : ideal R} :
+  I.jacobson = I ↔ ∃ M : set (ideal R), (∀ J ∈ M, is_maximal J ∨ J = ⊤) ∧ I = Inf M :=
```

Comment:
As above, I think it would be nice to also have the theorem in the form:
```lean
theorem eq_jacobson_iff_Inf_maximal' {I : ideal R} :
  I.jacobson = I ↔ ∃ M : set (ideal R), (∀ (J ∈ M) (K : ideal R), J < K → K = ⊤) ∧ I = Inf M :=
```